src/server/components/delete-modal.jsx

```diff
@@ -30,7 +32,9 @@ export default class DeleteModal extends React.Component {
     // add delete here
     try {
       await this.props.deleteFunction(this.props.elementID);
-      this.props.onClose();
+      if (this.props.onClose) {
+        this.props.onClose();
```

Comment:
if this prop (or onCloseOrCancel) is always required, we should not have to make this defensive check